src/components/DataSection/DataSection.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import './DataSection.scss';
+import { getLocation } from '../../commons/helpers';
+import Card from '../Card/Card';
+import { Tabs } from '../../statics/Statics';
+
+function DataSection ({activeTab}) {
+    const [data, setData] = useState([]);
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => getData(), [activeTab])
+
+    const callback = (results, status) => {
+     if (status === window.google.maps.places.PlacesServiceStatus.OK) {
+            setData(results)
+            setLoading(false)
+        }
+    }
+    const getData = () => {
+        setLoading(true)
+        getLocation().then(({ coords: {latitude, longitude} }) => {
+            var center = new window.google.maps.LatLng(latitude,longitude);
+            var map = new window.google.maps.Map(document.getElementById('map'), {
```

Comment:
Please cache the map based on the location. Creating a new map when location did not change can be expensive.